docker-compose.yml

```diff
@@ -33,6 +33,7 @@ services:
       OBJECTSTORAGE__SECRETKEY: ${OBJECTSTORAGE__SECRETKEY:-password}
       OBJECTSTORAGE__BUCKETNAME: ${OBJECTSTORAGE__ENDPOINT:-traffic-ticket-dev}
       OBJECTSTORAGE__SSL: ${OBJECTSTORAGE__SSL:-false}
+      HASHIDS__SALT: ${HASHIDS__SALT:-}
```

Comment:
Maybe update .env.template as well so we know this is a new required variable.

Reply:
good catch done